pkgs/servers/pingvin-share/frontend.nix

```diff
@@ -0,0 +1,47 @@
+{ version
+, src
+}:
```

Comment:
We don't need to create a function here and can just inherit version and src from the main package